### PR TITLE
add hackmd docs

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: docs/hardware/zeus.md [hunk 1]
```diff
@@ -13,3 +13,4 @@
 * [bitlbee](/services/bitlbee)
 * [git](/services/git)
 * [mastodon](/services/mastodon)
+* [hackmd](/services/hsckmd)
```
Comment (CSmartt) [inline docs/hardware/zeus.md]: hsckmd -> hackmd

File: docs/services/hackmd.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Hackmd
+
+Hackmd lives at [md.rb](https://md.redbrick.dcu.ie) on zeus
+
+Hackmd is build locally and is based on
```
Comment (CSmartt) [inline docs/services/hackmd.md]: build -> built

**Review by CSmartt — CHANGES_REQUESTED** [review]
Few spelling errors, grand other than that.

**Comment by butlerx** [comment]
ive fixed those
ive aslo added [sedy](https://marmelab.com/sedy/) for stuff like this going forward

**Review by CSmartt — APPROVED** [review]

**Review by VoyTechnology — COMMENTED** [review]
Idk about that capital letter ordering. Looks a bit out of place.

**Pull request merged**